tutorial_3_django_models/slides.md

```diff
@@ -0,0 +1,207 @@
+Tutorial: Create Django Models
+===============================
+[Demonstration time: 9 mins 25 secs (0.785 ~ 79%) | Total time: 12 mins]
+
+Slide 1 [00:08 | 00:08]
+-------------
+Title Slide: ** Creating Django Models **
+
+Slide 2 [00:12 | 00:20]
+--------------
+
+**Learning Objectives**
+
+In this tutorial, we will learn to;
+  - Create a django model
+  - Perform Database migration
+  - Use Admin app
+
+Slide 3 [00:11 | 00:31]
+---------------
+
+**System Requirements**
+  - Ubuntu 16.10
```

Comment:
Here we mention the software that we use for recording(i.e. this is spoken tutorial related and not django).
When showing this slide we say, "For recording this tutorial I am using..."
The end user can use any other version, but we stick to what we use because 
the end user is on same track as the recorded video.